generate-templates.yaml

```diff
@@ -374,3 +374,15 @@
     - {flavor: large,  workload: highperformance, memsize: "8Gi", cpus: 2, iothreads: True,  emulatorthread: True,  multiqueue: True,  tablet: True, default: False}
     vars:
       osinfoname: win10
+
+  - name: Generate Windows 11 templates
+    template:
+      src: windows11.tpl.yaml
+      dest: "{{ playbook_dir }}/dist/templates/windows11-{{ item.workload }}-{{ item.flavor }}.yaml"
+    with_items:
+    - {flavor: medium, workload: desktop,         memsize: "4Gi", cpus: 1, iothreads: False, emulatorthread: False, multiqueue: False, tablet: True, default: True}
+    - {flavor: medium, workload: highperformance, memsize: "4Gi", cpus: 1, iothreads: True,  emulatorthread: True,  multiqueue: True,  tablet: True, default: False}
```

Comment:
Windows 11 requires at least 2 cores: https://docs.microsoft.com/en-us/windows/whats-new/windows-11-requirements . Do we want to support singe core?

Reply:
I see, the template uses two cores, and this is just about the cpus.
@ksimon1 Should we align windows 10, too?

Reply:
Thanks for the review! Maybe I am confusing something but here I put the number of cpu. I put the minimum requirement of cores here
[https://github.com/kubevirt/common-templates/blob/e56147bc86cfdeffa638898ca15dea2a07fffc22/templates/windows11.tpl.yaml#L117](https://github.com/kubevirt/common-templates/blob/e56147bc86cfdeffa638898ca15dea2a07fffc22/templates/windows11.tpl.yaml#L117)
Am I missing something?

Reply:
Sorry I read now the second comment :(

Reply:
ok, thanks, let's keep it as proposed, and check how the new UI is going to handle it.
cc: @yaacov

Reply:
hi, sorry for the layman quastion
cpus and threads are per socket/cpu or per machine ?